packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -86,6 +86,18 @@ export interface TopicProps {
    * @default 1
    */
   readonly signatureVersion?: string;
+
+  /**
+   * Tracing mode of an Amazon SNS topic.
+   *
+   * If PassThrough, the topic passes trace headers received from the Amazon SNS publisher to its subscription.
+   * If set to Active, Amazon SNS will vend X-Ray segment data to topic owner account if the sampled flag in the tracing header is true.
+   *
+   * @see https://docs.aws.amazon.com/sns/latest/dg/sns-active-tracing.html.
```

Comment:
The period at the end breaks the URL

```suggestion
   * @see https://docs.aws.amazon.com/sns/latest/dg/sns-active-tracing.html
```